eng/Versions.props

```diff
@@ -1,11 +1,6 @@
 <Project>
   <!-- Import references updated by Dependabot. This file is for package references updated manually or by Maestro. -->
   <Import Project="dependabot/Versions.props" />
-  <Import Project="dependabot/nuget.org/Versions.props" />
```

Comment:
These are a one-time manual backport of #6071. Future automated merge PRs won't need to do this.